drc_cmis/webservice/request.py

```diff
@@ -104,15 +104,17 @@ def request(
         path: str,
         soap_envelope: str,
         attachments: Optional[List[Tuple[str, BinaryIO]]] = None,
-    ) -> str:
+        keep_binary: Optional[bool] = False,
```

Comment:
`Optional` means effectively `Union[None, <type>]` - in this case, we don't expect `None` as value as all.

```suggestion
        keep_binary: bool = False,
```